src/coprocessor/dag/rpn_expr/impl_cast.rs

```diff
@@ -55,6 +47,11 @@ pub fn get_cast_fn_rpn_node(
     })
 }
 
+#[inline]
+fn is_unsigned(ft: &FieldType) -> bool {
```

Comment:
why not a FieldType method?

Reply:
The FieldType is protobuf type, we cannot attach a method to it.

Reply:
Then how about in `FieldTypeAccessor`?

Reply:
@breeswish https://github.com/tikv/tikv/pull/5095